canonical-web-and-design/ubuntu.com
# Build new /security/docker-images page

## Done

- Build new `/security/docker-images` page 
- Add contact us and thank you pages for Docker Images
- Add to secondary nav and remove 'Overview' 

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/security/docker-images
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check page against [copy doc](https://docs.google.com/document/d/1zZibtjU141e4mBxo_0PtfYxJ6e5fyaaAdFZ6v945XAM/edit#)


## Issue / Card

Fixes [#3492](https://github.com/canonical-web-and-design/web-squad/issues/3492)
